common4j/src/main/com/microsoft/identity/common/java/controllers/BaseController.java

```diff
@@ -312,6 +312,12 @@ protected final AuthorizationRequest.Builder initializeAuthorizationRequestBuild
             );
             completeRequestHeaders.put(PKEYAUTH_HEADER, PKEYAUTH_VERSION);
 
+            // Not going to add passkey protocol header until full feature is ready.
+            if (FidoConstants.IS_PASSKEY_SUPPORT_READY
+                    && interactiveTokenCommandParameters.getAuthorizationAgent() == AuthorizationAgent.WEBVIEW) {
+                completeRequestHeaders.put(FidoConstants.PASSKEY_PROTOCOL_HEADER_NAME, FidoConstants.PASSKEY_PROTOCOL_HEADER_VALUE);
+            }
```

Comment:
we need to make similar change in PrtV3AuthorizationStrategy as well, as it creates auth request for acquire PRTv3 (sorry to point this late.)

I see potential to refactor this code out to OAuthStrategy and it's derived classes and then won't have that redundancy. We can do that as separate change.

Reply:
See this PR: https://github.com/AzureAD/ad-accounts-for-android/pull/2540